timelock-agent/src/main/java/com/palantir/atlasdb/timelock/paxos/SingleLeaderNetworkClientFactories.java

```diff
@@ -26,21 +27,25 @@
 import com.palantir.paxos.PaxosLearnerNetworkClient;
 import com.palantir.paxos.SingleLeaderAcceptorNetworkClient;
 import com.palantir.paxos.SingleLeaderLearnerNetworkClient;
-import com.palantir.timelock.paxos.TimelockPaxosAcceptorAdapter;
-import com.palantir.timelock.paxos.TimelockPaxosLearnerAdapter;
+import com.palantir.timelock.paxos.TimelockPaxosAcceptorAdapters;
+import com.palantir.timelock.paxos.TimelockPaxosLearnerAdapters;
 
 @Value.Immutable
 abstract class SingleLeaderNetworkClientFactories implements
         NetworkClientFactories, Dependencies.NetworkClientFactories {
 
+    @Value.Default
+    Supplier<Boolean> useBatchedEndpoints() {
```

Comment:
uncertain: Might be better to use primitive `BooleanSupplier`, though I wouldn't block if you need `Supplier<T>` for something down the line.

Reply:
PredicateSwitchedProxy uses `Supplier<Boolean>` so just more consistent